src/main/kotlin/string/Calculator.kt

```diff
@@ -0,0 +1,58 @@
+package string
+
+class Calculator() {
+    fun sum(input: String?): Int {
+        if (input.isNullOrBlank()) {
+            return 0
+        }
+
+        return add(parse(input))
```

Comment:
바로 return 후 if 식을 사용해볼 수도 있겠네요 🙂
```suggestion
        return if (input.isNullOrBlank()) 0
        else add(parse(input))
```